Hardcoded legacy johnlanni/openclaw repo in Manager Dockerfiles causes TypeScript errors

### Bug Description

Several Dockerfiles in the manager/ directory (specifically Dockerfile.copaw and Dockerfile.aliyun) are hardcoded to clone an outdated, legacy fork of the OpenClaw engine instead of the official upstream repository. This pulls in severely outdated code that triggers TypeScript errors during the UI build phase and prevents the use of google models (like Gemini 3), without needing brittle patches and wrappers instead of upstream fixes from openclaw.
Additionally 

Proposed Solution

Update the git clone commands in all manager/Dockerfile.* variants to point to openclaw/openclaw.

Add a --default-timeout=120 flag to the pip install commands in these Dockerfiles to prevent brittle network timeouts when downloading large packages from the Aliyun mirror.

### Steps to Reproduce

1. Attempt to build hiclaw/manager-copaw or hiclaw/manager-aliyun using the provided Dockerfiles.

2. The build executes the following line:
git clone --depth 1 --single-branch -b hiclaw-v1 https://github.com/johnlanni/openclaw.git /opt/openclaw

3. During the npm run build (Vite) phase, the compiler throws hundreds of type errors (e.g., error TS2580: Cannot find name 'process', error TS1360).

4. Additionally, the hardcoded Aliyun mirror (PIP_INDEX_URL=https://mirrors.aliyun.com/pypi/simple/) frequently causes ReadTimeoutError during the copaw and matrix-nio pip installations.

The Dockerfiles should clone from the official upstream repository (https://github.com/openclaw/openclaw.git) using a stable release tag, rather than a stale hiclaw-v1 branch from a personal fork.

### AI Analysis

The Dockerfiles should clone from the official upstream repository (https://github.com/openclaw/openclaw.git) using a stable release tag or the main branch, rather than a stale hiclaw-v1 branch from a personal fork.

### Relevant Logs

```shell
==> Building OpenClaw base image: hiclaw/openclaw-base:latest (registry: higress-registry.cn-hangzhou.cr.aliyuncs.com)
docker build  --build-arg HIGRESS_REGISTRY=higress-registry.cn-hangzhou.cr.aliyuncs.com  \
	-t hiclaw/openclaw-base:latest \
	./openclaw-base/
STEP 1/13: FROM higress-registry.cn-hangzhou.cr.aliyuncs.com/higress/all-in-one:sha-d32debd
STEP 2/13: ARG APT_MIRROR=mirrors.aliyun.com
--> Using cache 6c48d1a2b552a931f2993aa56a47e9bea6327c79db2f490c25341976b59caa87
--> 6c48d1a2b552
STEP 3/13: ARG HTTP_PROXY
--> Using cache 0e0525a486e7b92d9ff6c8bfca7c7cb80011df382a7a48a2b274ee8fa4e89b17
--> 0e0525a486e7
STEP 4/13: ARG HTTPS_PROXY
--> Using cache 2ab510bb6bf5b88308a5ab152f9b113260e22e30c4b26335284204cba85b99cf
--> 2ab510bb6bf5
STEP 5/13: ARG http_proxy
--> Using cache a8c8a5660d2edae6833aaf01e12793cc622b944c093ec808caaee2b42fdcdc9a
--> a8c8a5660d2e
STEP 6/13: ARG https_proxy
--> Using cache 1add668dc0dce31d9fa9a9a02fb0a481ff18d78f9b5d244f9f5c9062db29d027
--> 1add668dc0dc
STEP 7/13: ARG NPM_REGISTRY=https://registry.npmjs.org/
--> Using cache 273b5a575880491a36371916834ff9d1927455e078c869b90b22db8849311e8a
--> 273b5a575880
STEP 8/13: ENV HTTP_PROXY=${HTTP_PROXY}     HTTPS_PROXY=${HTTPS_PROXY}     http_proxy=${http_proxy}     https_proxy=${https_proxy}
--> Using cache 2ec7e3235bf509e850f670c69e4d8223a4e184ad772a2342848542a9c6ef291c
--> 2ec7e3235bf5
STEP 9/13: RUN if [ -n "${APT_MIRROR}" ]; then         sed -i "s|archive.ubuntu.com|${APT_MIRROR}|g; s|security.ubuntu.com|${APT_MIRROR}|g"             /etc/apt/sources.list 2>/dev/null || true;     fi &&     apt-get update && apt-get install -y     git python3 make g++ curl     jq nginx gettext-base openssh-client ca-certificates procps tzdata     && rm -rf /var/lib/apt/lists/*
--> Using cache e5a7081c36eb5101bb2a71e73c898f027a95a49e2092fea1671709ab307939a6
--> e5a7081c36eb
STEP 10/13: RUN curl -fsSL https://deb.nodesource.com/setup_22.x | bash - &&     apt-get install -y nodejs &&     npm install -g pnpm &&     rm -rf /var/lib/apt/lists/*
--> Using cache 74e798acd0f3fcd38bff48efbdde92925ccbb4251f1141dacd9a10afac5805d6
--> 74e798acd0f3
STEP 11/13: RUN git clone --depth 1 --single-branch -b hiclaw-v1         https://github.com/johnlanni/openclaw.git /opt/openclaw &&     cd /opt/openclaw &&     git fetch --depth 1 origin 86dad1383c12629ad3aa48575f2dd350fc0775d4 &&     git checkout 86dad1383c12629ad3aa48575f2dd350fc0775d4 &&     pnpm config set registry ${NPM_REGISTRY} &&     pnpm install &&     pnpm build &&     pnpm ui:build &&     echo "==> Verifying critical native addons..." &&     ADDON_DIR=/opt/openclaw/node_modules/.pnpm/@matrix-org+matrix-sdk-crypto-nodejs@0.4.0/node_modules/@matrix-org/matrix-sdk-crypto-nodejs &&     find /opt/openclaw/node_modules -name "matrix-sdk-crypto.*.node" 2>/dev/null | grep -q .         || (echo "ERROR: matrix-sdk-crypto native addon missing (pnpm install may have silently failed on this arch)" && exit 1) &&     echo "==> Native addon check passed: $(ls $ADDON_DIR/matrix-sdk-crypto.linux-*.node)" &&     rm -rf         .git .github         test         docker-compose.yml Dockerfile*         scripts/git-hooks         assets README-header.png         .secrets.baseline .pre-commit-config.yaml         .dockerignore .gitignore .gitattributes         .oxfmtrc.jsonc .oxlintrc.json .shellcheckrc         .detect-secrets.cfg .npmrc         .env.example appcast.xml         vitest*.config.ts tsdown.config.ts fly*.toml render.yaml &&     rm -rf /root/.local/share/pnpm /root/.npm
Cloning into '/opt/openclaw'...
Updating files:  45% (3667/8060)
Updating files:  46% (3708/8060)
Updating files:  47% (3789/8060)
Updating files:  48% (3869/8060)
Updating files:  49% (3950/8060)
Updating files:  50% (4030/8060)
Updating files:  51% (4111/8060)
Updating files:  52% (4192/8060)
Updating files:  53% (4272/8060)
Updating files:  54% (4353/8060)
Updating files:  55% (4433/8060)
Updating files:  56% (4514/8060)
Updating files:  57% (4595/8060)
Updating files:  58% (4675/8060)
Updating files:  59% (4756/8060)
Updating files:  60% (4836/8060)
Updating files:  61% (4917/8060)
Updating files:  62% (4998/8060)
Updating files:  63% (5078/8060)
Updating files:  64% (5159/8060)
Updating files:  65% (5239/8060)
Updating files:  66% (5320/8060)
Updating files:  67% (5401/8060)
Updating files:  68% (5481/8060)
Updating files:  69% (5562/8060)
Updating files:  70% (5642/8060)
Updating files:  71% (5723/8060)
Updating files:  72% (5804/8060)
Updating files:  73% (5884/8060)
Updating files:  74% (5965/8060)
Updating files:  75% (6045/8060)
Updating files:  76% (6126/8060)
Updating files:  77% (6207/8060)
Updating files:  78% (6287/8060)
Updating files:  79% (6368/8060)
Updating files:  80% (6448/8060)
Updating files:  81% (6529/8060)
Updating files:  82% (6610/8060)
Updating files:  83% (6690/8060)
Updating files:  84% (6771/8060)
Updating files:  85% (6851/8060)
Updating files:  86% (6932/8060)
Updating files:  87% (7013/8060)
Updating files:  88% (7093/8060)
Updating files:  89% (7174/8060)
Updating files:  90% (7254/8060)
Updating files:  91% (7335/8060)
Updating files:  92% (7416/8060)
Updating files:  93% (7496/8060)
Updating files:  94% (7577/8060)
Updating files:  95% (7657/8060)
Updating files:  96% (7738/8060)
Updating files:  96% (7788/8060)
Updating files:  97% (7819/8060)
Updating files:  98% (7899/8060)
Updating files:  99% (7980/8060)
Updating files: 100% (8060/8060)
Updating files: 100% (8060/8060), done.
From https://github.com/johnlanni/openclaw
 * branch            86dad1383c12629ad3aa48575f2dd350fc0775d4 -> FETCH_HEAD
Note: switching to '86dad1383c12629ad3aa48575f2dd350fc0775d4'.
You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.
If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:
  git switch -c <new-branch-name>
Or undo this operation with:
  git switch -
Turn off this advice by setting config variable advice.detachedHead to false
HEAD is now at 86dad13 fix(matrix): include media info in history entries for non-mentioned messages
Scope: all 37 workspace projects
Lockfile is up to date, resolution step is skipped
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +1263
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 1263, reused 0, downloaded 7, added 0
Progress: resolved 1263, reused 0, downloaded 23, added 0
Progress: resolved 1263, reused 0, downloaded 72, added 20
Progress: resolved 1263, reused 0, downloaded 129, added 42
Progress: resolved 1263, reused 0, downloaded 220, added 79
Progress: resolved 1263, reused 0, downloaded 274, added 132
Progress: resolved 1263, reused 0, downloaded 348, added 171
Progress: resolved 1263, reused 0, downloaded 370, added 183
Progress: resolved 1263, reused 0, downloaded 387, added 188
Progress: resolved 1263, reused 0, downloaded 413, added 197
Progress: resolved 1263, reused 0, downloaded 473, added 270
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /root/.local/share/pnpm/store/v10
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 1263, reused 0, downloaded 571, added 336
Progress: resolved 1263, reused 0, downloaded 671, added 395
Progress: resolved 1263, reused 0, downloaded 725, added 444
Progress: resolved 1263, reused 0, downloaded 767, added 505
Progress: resolved 1263, reused 0, downloaded 779, added 508
Progress: resolved 1263, reused 0, downloaded 789, added 513
Progress: resolved 1263, reused 0, downloaded 824, added 530
Progress: resolved 1263, reused 0, downloaded 844, added 535
Progress: resolved 1263, reused 0, downloaded 861, added 542
Progress: resolved 1263, reused 0, downloaded 891, added 556
Progress: resolved 1263, reused 0, downloaded 913, added 565
Progress: resolved 1263, reused 0, downloaded 923, added 569
Progress: resolved 1263, reused 0, downloaded 933, added 573
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install$ npm install
Progress: resolved 1263, reused 0, downloaded 938, added 576
Progress: resolved 1263, reused 0, downloaded 939, added 576
Progress: resolved 1263, reused 0, downloaded 961, added 584
Progress: resolved 1263, reused 0, downloaded 964, added 585
Progress: resolved 1263, reused 0, downloaded 975, added 589
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: npm warn deprecated lodash.get@4.4.2: This package is deprecated. Use the optional chaining (?.) operator instead.
Progress: resolved 1263, reused 0, downloaded 981, added 592
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: npm warn deprecated lodash.isequal@4.5.0: This package is deprecated. Use require('node:util').isDeepStrictEqual instead.
Progress: resolved 1263, reused 0, downloaded 988, added 596
Progress: resolved 1263, reused 0, downloaded 998, added 597
Progress: resolved 1263, reused 0, downloaded 1010, added 601
Progress: resolved 1263, reused 0, downloaded 1019, added 605
Progress: resolved 1263, reused 0, downloaded 1075, added 645
Progress: resolved 1263, reused 0, downloaded 1157, added 722
Progress: resolved 1263, reused 0, downloaded 1218, added 773
Progress: resolved 1263, reused 0, downloaded 1259, added 915
Progress: resolved 1263, reused 0, downloaded 1259, added 1261
Progress: resolved 1263, reused 0, downloaded 1259, added 1262
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: > @tloncorp/api@0.0.2 prepare
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: > npm run build
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: > @tloncorp/api@0.0.2 build
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: > vite build
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: vite v5.4.21 building for production...
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/api/channelContentConfig.ts:113:12 - error TS1360: Type '{ readonly 'tlon.r0.collection.chat': { readonly displayName: "Chat"; readonly enumTag: "chat"; readonly parametersSchema: Record<string, ParameterSpec>; }; ... 7 more ...; readonly 'tlon.r0.collection.summaries': { ...; }; }' does not satisfy the expected type 'Record<string, ComponentSpec<string>>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property ''tlon.r0.collection.carousel'' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     Type '{ readonly displayName: "Carousel"; readonly enumTag: "carousel"; readonly parametersSchema: { readonly scrollDirection: { readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ ...; }, { ...; }]; }; }; }' is not assignable to type 'ComponentSpec<string>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:       Types of property 'parametersSchema' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:         Property 'scrollDirection' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:           Type '{ readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]; }' is not assignable to type 'ParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:             Types of property 'options' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:               The type 'readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]' is 'readonly' and cannot be assigned to the mutable type '{ displayName: string; value: string; }[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 113 } as const satisfies Record<string, ComponentSpec>;
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                ~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/api/channelContentConfig.ts:207:12 - error TS1360: Type '{ readonly 'tlon.r0.content.chat': { readonly displayName: "Chat"; readonly enumTag: "chat"; }; readonly 'tlon.r0.content.gallery': { readonly displayName: "Gallery"; readonly enumTag: "gallery"; readonly parametersSchema: { ...; }; }; ... 5 more ...; readonly 'tlon.r0.content.scratchpad': { ...; }; }' does not satisfy the expected type 'Record<string, ComponentSpec<string>>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property ''tlon.r0.content.gallery'' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     Type '{ readonly displayName: "Gallery"; readonly enumTag: "gallery"; readonly parametersSchema: { readonly embedded: { readonly displayName: "Show frame"; readonly type: "boolean"; }; readonly contentSize: { ...; }; }; }' is not assignable to type 'ComponentSpec<string>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:       Types of property 'parametersSchema' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:         Property 'contentSize' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:           Type '{ readonly displayName: "Content size"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { readonly displayName: "Small"; readonly value: "$s"; }]; }' is not assignable to type 'ParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:             Types of property 'options' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:               The type 'readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { readonly displayName: "Small"; readonly value: "$s"; }]' is 'readonly' and cannot be assigned to the mutable type '{ displayName: string; value: string; }[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 207 } as const satisfies Record<string, ComponentSpec>;
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                ~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/api/channelContentConfig.ts:209:46 - error TS2345: Argument of type '{ readonly 'tlon.r0.collection.chat': { readonly displayName: "Chat"; readonly enumTag: "chat"; readonly parametersSchema: Record<string, ParameterSpec>; }; ... 7 more ...; readonly 'tlon.r0.collection.summaries': { ...; }; }' is not assignable to parameter of type 'Record<string, ComponentSpec<string>>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property ''tlon.r0.collection.carousel'' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     Type '{ readonly displayName: "Carousel"; readonly enumTag: "carousel"; readonly parametersSchema: { readonly scrollDirection: { readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ ...; }, { ...; }]; }; }; }' is not assignable to type 'ComponentSpec<string>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:       Types of property 'parametersSchema' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:         Type '{ readonly scrollDirection: { readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]; }; }' is not assignable to type 'Record<string, ParameterSpec>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:           Property 'scrollDirection' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:             Type '{ readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]; }' is not assignable to type 'ParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:               Type '{ readonly displayName: "Scroll direction"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]; }' is not assignable to type 'RadioParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                 Types of property 'options' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                   The type 'readonly [{ readonly displayName: "Horizontal"; readonly value: "horizontal"; }, { readonly displayName: "Vertical"; readonly value: "vertical"; }]' is 'readonly' and cannot be assigned to the mutable type '{ displayName: string; value: string; }[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 209 export const CollectionRendererId = makeEnum(allCollectionRenderers);
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                  ~~~~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/api/channelContentConfig.ts:215:47 - error TS2345: Argument of type '{ readonly 'tlon.r0.content.chat': { readonly displayName: "Chat"; readonly enumTag: "chat"; }; readonly 'tlon.r0.content.gallery': { readonly displayName: "Gallery"; readonly enumTag: "gallery"; readonly parametersSchema: { ...; }; }; ... 5 more ...; readonly 'tlon.r0.content.scratchpad': { ...; }; }' is not assignable to parameter of type 'Record<string, ComponentSpec<string>>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property ''tlon.r0.content.gallery'' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     Type '{ readonly displayName: "Gallery"; readonly enumTag: "gallery"; readonly parametersSchema: { readonly embedded: { readonly displayName: "Show frame"; readonly type: "boolean"; }; readonly contentSize: { ...; }; }; }' is not assignable to type 'ComponentSpec<string>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:       Types of property 'parametersSchema' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:         Type '{ readonly embedded: { readonly displayName: "Show frame"; readonly type: "boolean"; }; readonly contentSize: { readonly displayName: "Content size"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { ...; }]; }; }' is not assignable to type 'Record<string, ParameterSpec>'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:           Property 'contentSize' is incompatible with index signature.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:             Type '{ readonly displayName: "Content size"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { readonly displayName: "Small"; readonly value: "$s"; }]; }' is not assignable to type 'ParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:               Type '{ readonly displayName: "Content size"; readonly type: "radio"; readonly options: readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { readonly displayName: "Small"; readonly value: "$s"; }]; }' is not assignable to type 'RadioParameterSpec'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                 Types of property 'options' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                   The type 'readonly [{ readonly displayName: "Large"; readonly value: "$l"; }, { readonly displayName: "Small"; readonly value: "$s"; }]' is 'readonly' and cannot be assigned to the mutable type '{ displayName: string; value: string; }[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 215 export const PostContentRendererId = makeEnum(allContentRenderers);
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                   ~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/debug.ts:13:14 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 13   if (typeof process !== 'undefined' && process.env) {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                 ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/debug.ts:13:41 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 13   if (typeof process !== 'undefined' && process.env) {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                            ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/debug.ts:14:12 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 14     return process.env.NODE_ENV !== 'production';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:               ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/debug.ts:53:7 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 53   if (process.env.NODE_ENV !== 'production') {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:          ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/constants.ts:90:14 - error TS2304: Cannot find name 'global'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 90   if (typeof global !== 'undefined' && (global as any)[TLON_NAMESPACE]) {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                 ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/constants.ts:90:41 - error TS2304: Cannot find name 'global'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 90   if (typeof global !== 'undefined' && (global as any)[TLON_NAMESPACE]) {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                            ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/constants.ts:91:40 - error TS2304: Cannot find name 'global'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 91     return { ...DEFAULT_CONSTANTS, ...(global as any)[TLON_NAMESPACE] };
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                           ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/constants.ts:104:21 - error TS2304: Cannot find name 'global'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 104   } else if (typeof global !== 'undefined') {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                         ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/constants.ts:105:6 - error TS2304: Cannot find name 'global'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 105     (global as any)[TLON_NAMESPACE] = { ...getConstants(), ...constants };
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:          ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/groupTemplates.ts:163:12 - error TS1360: Type 'readonly [{ readonly id: "book-club"; readonly title: "Book Club"; readonly subtitle: "Discuss your latest reads"; readonly description: "A group for discussing books and literature"; readonly icon: "📚"; readonly channels: readonly [...]; }, ... 4 more ..., { ...; }]' does not satisfy the expected type 'GroupTemplate[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   The type 'readonly [{ readonly id: "book-club"; readonly title: "Book Club"; readonly subtitle: "Discuss your latest reads"; readonly description: "A group for discussing books and literature"; readonly icon: "📚"; readonly channels: readonly [...]; }, ... 4 more ..., { ...; }]' is 'readonly' and cannot be assigned to the mutable type 'GroupTemplate[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 163 ] as const satisfies GroupTemplate[];
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                ~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/groupTemplates.ts:189:12 - error TS1360: Type '{ readonly id: "basic-group"; readonly title: "Basic Group"; readonly subtitle: "A basic group with essential channels"; readonly description: "A basic group with essential channels"; readonly icon: "✨"; readonly channels: readonly [...]; }' does not satisfy the expected type 'GroupTemplate'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Types of property 'channels' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     The type 'readonly [{ readonly type: "chat"; readonly title: "Chat"; readonly description: "General chat"; }, { readonly type: "gallery"; readonly title: "Gallery"; readonly description: "Share images"; }, { ...; }]' is 'readonly' and cannot be assigned to the mutable type 'TemplateChannel[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 189 } as const satisfies GroupTemplate;
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                ~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/domain/groupTemplates.ts:197:5 - error TS2322: Type '{ readonly id: "book-club"; readonly title: "Book Club"; readonly subtitle: "Discuss your latest reads"; readonly description: "A group for discussing books and literature"; readonly icon: "📚"; readonly channels: readonly [...]; } | ... 5 more ... | { ...; }' is not assignable to type 'GroupTemplate'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Type '{ readonly id: "book-club"; readonly title: "Book Club"; readonly subtitle: "Discuss your latest reads"; readonly description: "A group for discussing books and literature"; readonly icon: "📚"; readonly channels: readonly [...]; }' is not assignable to type 'GroupTemplate'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:     Types of property 'channels' are incompatible.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:       The type 'readonly [{ readonly type: "chat"; readonly title: "Book chat"; readonly description: "Discuss books and literature"; }, { readonly type: "gallery"; readonly title: "Now reading"; readonly description: "Share what you're currently reading"; }, { ...; }]' is 'readonly' and cannot be assigned to the mutable type 'TemplateChannel[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 197     acc[template.id as GroupTemplateId] = template;
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/embed.ts:1:26 - error TS2307: Cannot find module '@tanstack/react-query' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 1 import { useQuery } from '@tanstack/react-query';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                            ~~~~~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/embed.ts:2:29 - error TS2307: Cannot find module 'react' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 2 import { useCallback } from 'react';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                               ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:454:34 - error TS2339: Property 'image' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'image' does not exist on type '{ cite: Cite; }'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 454       if (VIDEO_REGEX.test(block.image.src)) {
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                      ~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:457:20 - error TS2339: Property 'image' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'image' does not exist on type '{ cite: Cite; }'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 457           ...block.image,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                        ~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:462:20 - error TS2339: Property 'image' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'image' does not exist on type '{ cite: Cite; }'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 462           ...block.image,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                        ~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:470:36 - error TS2339: Property 'listing' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'listing' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 470         list: convertListing(block.listing),
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                        ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:476:22 - error TS2339: Property 'header' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'header' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 476         level: block.header.tag,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                          ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:477:46 - error TS2339: Property 'header' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'header' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 477         children: convertInlineContent(block.header.content),
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                  ~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:483:24 - error TS2339: Property 'code' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'code' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 483         content: block.code.code,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                            ~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:484:21 - error TS2339: Property 'code' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'code' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 484         lang: block.code.lang,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                         ~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:494:38 - error TS2339: Property 'cite' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'cite' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 494         api.toContentReference(block.cite) ?? errorMessage('Failed to parse')
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                          ~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:499:18 - error TS2339: Property 'link' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'link' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 499         ...block.link.meta,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                      ~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:501:20 - error TS2339: Property 'link' does not exist on type 'Block'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Property 'link' does not exist on type 'Image'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 501         url: block.link.url,
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                        ~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/postContent.ts:505:19 - error TS2345: Argument of type 'Block' is not assignable to parameter of type 'never'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   Type 'Image' is not assignable to type 'never'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 505       assertNever(block);
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                       ~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/utilHooks.ts:1:34 - error TS2307: Cannot find module '@tanstack/react-query' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 1 import { replaceEqualDeep } from '@tanstack/react-query';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                    ~~~~~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/utilHooks.ts:2:67 - error TS2307: Cannot find module 'react' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 2 import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                                     ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/logic/utilHooks.ts:61:55 - error TS2345: Argument of type 'any[]' is not assignable to parameter of type 'Args'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   'any[]' is assignable to the constraint of type 'Args', but 'Args' could be instantiated with a different subtype of constraint 'unknown[]'.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 61   return useCallback((...args) => callbackRef.current(...args), [callbackRef]);
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                          ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/store/reactQuery.ts:1:50 - error TS2307: Cannot find module '@tanstack/react-query' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 1 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                    ~~~~~~~~~~~~~~~~~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: src/utils/useEventEmitter.ts:1:50 - error TS2307: Cannot find module 'react' or its corresponding type declarations.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 1 import { useCallback, useEffect, useState } from 'react';
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:                                                    ~~~~~~~
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: transforming...
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: ✓ 1293 modules transformed.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: rendering chunks...
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: [vite:dts] Start generate declaration files...
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: computing gzip size...
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: dist/index.js  774.23 kB │ gzip: 200.21 kB │ map: 2,919.47 kB
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: [vite:dts] Declaration files built in 4549ms.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: dist/index.cjs  549.76 kB │ gzip: 174.16 kB │ map: 2,855.95 kB
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: ✓ built in 9.80s
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: added 277 packages, and audited 278 packages in 33s
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 41 packages are looking for funding
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   run `npm fund` for details
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: 19 vulnerabilities (11 moderate, 6 high, 2 critical)
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: To address issues that do not require attention, run:
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   npm audit fix
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: To address all issues (including breaking changes), run:
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install:   npm audit fix --force
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: Run `npm audit` for details.
..._114_c71f4f004a2f07b5210e67586fe7c6e9 npm-install: Done
Progress: resolved 1263, reused 0, downloaded 1260, added 1262
Progress: resolved 1263, reused 0, downloaded 1260, added 1263, done
.../koffi@2.15.1/node_modules/koffi install$ node src/cnoke/cnoke.js -p . -d src/koffi --prebuild
.../node_modules/protobufjs postinstall$ node scripts/postinstall
.../esbuild@0.27.3/node_modules/esbuild postinstall$ node install.js
.../node_modules/protobufjs postinstall$ node scripts/postinstall
.../koffi@2.15.1/node_modules/koffi install: Done
.../node_modules/protobufjs postinstall: Done
.../esbuild@0.27.3/node_modules/esbuild postinstall: Done
.../sharp@0.34.5/node_modules/sharp install$ node install/check.js || npm run build
.../node_modules/protobufjs postinstall: Done
.../node_modules/node-llama-cpp postinstall$ node ./dist/cli/cli.js postinstall
.../@matrix-org/matrix-sdk-crypto-nodejs postinstall$ node download-lib.js
.../sharp@0.34.5/node_modules/sharp install: Done
.../@matrix-org/matrix-sdk-crypto-nodejs postinstall: Downloading lib matrix-sdk-crypto.linux-x64-gnu.node from https://github.com/matrix-org/matrix-rust-sdk-crypto-nodejs/releases/download/v0.4.0/matrix-sdk-crypto.linux-x64-gnu.node
.../@matrix-org/matrix-sdk-crypto-nodejs postinstall: Download Completed
.../@matrix-org/matrix-sdk-crypto-nodejs postinstall: Done
.../node_modules/node-llama-cpp postinstall: Done
.../node_modules/@whiskeysockets/baileys preinstall$ node ./engine-requirements.js
.../node_modules/@whiskeysockets/baileys preinstall: Done
dependencies:
+ @agentclientprotocol/sdk 0.15.0
+ @aws-sdk/client-bedrock 3.1004.0
+ @buape/carbon 0.0.0-beta-20260216184201
+ @clack/prompts 1.1.0
+ @discordjs/voice 0.19.0
+ @grammyjs/runner 2.0.3
+ @grammyjs/transformer-throttler 1.2.1
+ @homebridge/ciao 1.3.5
+ @larksuiteoapi/node-sdk 1.59.0
+ @line/bot-sdk 10.6.0
+ @lydell/node-pty 1.2.0-beta.3
+ @mariozechner/pi-agent-core 0.57.1
+ @mariozechner/pi-ai 0.57.1
+ @mariozechner/pi-coding-agent 0.57.1
+ @mariozechner/pi-tui 0.57.1
+ @mozilla/readability 0.6.0
+ @napi-rs/canvas 0.1.95
+ @sinclair/typebox 0.34.48
+ @slack/bolt 4.6.0
+ @slack/web-api 7.14.1
+ @whiskeysockets/baileys 7.0.0-rc.9
+ ajv 8.18.0
+ chalk 5.6.2
+ chokidar 5.0.0
+ cli-highlight 2.1.11
+ commander 14.0.3
+ croner 10.0.1
+ discord-api-types 0.38.41
+ dotenv 17.3.1
+ express 5.2.1
+ file-type 21.3.0
+ grammy 1.41.1
+ https-proxy-agent 7.0.6
+ ipaddr.js 2.3.0
+ jiti 2.6.1
+ json5 2.2.3
+ jszip 3.10.1
+ linkedom 0.18.12
+ long 5.3.2
+ markdown-it 14.1.1
+ node-edge-tts 1.2.10
+ node-llama-cpp 3.16.2
+ opusscript 0.1.1
+ osc-progress 0.3.0
+ pdfjs-dist 5.5.207
+ playwright-core 1.58.2
+ qrcode-terminal 0.12.0
+ sharp 0.34.5
+ sqlite-vec 0.1.7-alpha.2
+ tar 7.5.10
+ tslog 4.10.2
+ undici 7.22.0
+ ws 8.19.0
+ yaml 2.8.2
+ zod 4.3.6
devDependencies:
+ @grammyjs/types 3.25.0
+ @lit-labs/signals 0.2.0
+ @lit/context 1.1.6
+ @types/express 5.0.6
+ @types/markdown-it 14.1.2
+ @types/node 25.3.5
+ @types/qrcode-terminal 0.12.2
+ @types/ws 8.18.1
+ @typescript/native-preview 7.0.0-dev.20260308.1
+ @vitest/coverage-v8 4.0.18
+ jscpd 4.0.8
+ lit 3.3.2
+ oxfmt 0.36.0
+ oxlint 1.51.0
+ oxlint-tsgolint 0.16.0
+ signal-utils 0.21.1
+ tsdown 0.21.0
+ tsx 4.21.0
+ typescript 5.9.3
+ vitest 4.0.18
╭ Warning ─────────────────────────────────────────────────────────────────────╮
│                                                                              │
│   Ignored build scripts: @discordjs/opus, @tloncorp/tlon-skill.              │
│   Run "pnpm approve-builds" to pick which dependencies should be allowed     │
│   to run scripts.                                                            │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
. prepare$ command -v git >/dev/null 2>&1 && git rev-parse --is-inside-work-tree >/dev/null 2>&1 && git config core.hooksPath git-hooks || exit 0
. prepare: Done
Done in 1m 8.6s using pnpm v10.23.0
> openclaw@2026.3.8 build /opt/openclaw
> pnpm canvas:a2ui:bundle && node scripts/tsdown-build.mjs && node scripts/copy-plugin-sdk-root-alias.mjs && pnpm build:plugin-sdk:dts && node --import tsx scripts/write-plugin-sdk-entry-dts.ts && node --import tsx scripts/canvas-a2ui-copy.ts && node --import tsx scripts/copy-hook-metadata.ts && node --import tsx scripts/copy-export-html-templates.ts && node --import tsx scripts/write-build-info.ts && node --import tsx scripts/write-cli-startup-metadata.ts && node --import tsx scripts/write-cli-compat.ts
> openclaw@2026.3.8 canvas:a2ui:bundle /opt/openclaw
> bash scripts/bundle-a2ui.sh
<DIR>/a2ui.bundle.js  chunk │ size: 486.00 kB
✔ rolldown v1.0.0-rc.15 Finished in 115.89 ms
> openclaw@2026.3.8 build:plugin-sdk:dts /opt/openclaw
> tsc -p tsconfig.plugin-sdk.dts.json
[copy-hook-metadata] Copied 4 hook metadata files.
[copy-export-html-templates] Copied 5 export-html assets.
> openclaw@2026.3.8 ui:build /opt/openclaw
> node scripts/ui.js build
> openclaw-control-ui@ build /opt/openclaw/ui
> vite build
vite v7.3.1 building client environment for production...
transforming...
✓ 185 modules transformed.
rendering chunks...
computing gzip size...
../dist/control-ui/index.html                   0.69 kB │ gzip:   0.37 kB
../dist/control-ui/assets/index-E0j6Tkrc.css   94.25 kB │ gzip:  16.32 kB
../dist/control-ui/assets/zh-TW-cW5xB87I.js     3.69 kB │ gzip:   2.22 kB │ map:     6.16 kB
../dist/control-ui/assets/pt-BR-D2dJb9G8.js     4.16 kB │ gzip:   2.03 kB │ map:     6.69 kB
../dist/control-ui/assets/de-DuUYLvt1.js        4.31 kB │ gzip:   2.11 kB │ map:     6.87 kB
../dist/control-ui/assets/zh-CN-BgJ8_lE3.js     9.94 kB │ gzip:   4.85 kB │ map:    16.78 kB
../dist/control-ui/assets/es-DHtyqUQZ.js       11.92 kB │ gzip:   4.79 kB │ map:    19.06 kB
../dist/control-ui/assets/index-wxM3V0HM.js   657.64 kB │ gzip: 166.77 kB │ map: 1,847.57 kB
✓ built in 2.44s
==> Verifying critical native addons...
==> Native addon check passed: /opt/openclaw/node_modules/.pnpm/@matrix-org+matrix-sdk-crypto-nodejs@0.4.0/node_modules/@matrix-org/matrix-sdk-crypto-nodejs/matrix-sdk-crypto.linux-x64-gnu.node
--> 16d44538ba75
STEP 12/13: ...
```

### Component

Install / Setup

### Version / Commit

v1.0.9

--- a/openclaw-base/Dockerfile
+++ b/openclaw-base/Dockerfile
@@ -10,6 +10,7 @@
 # Build args:
 #   HIGRESS_REGISTRY  - base image registry (default: cn-hangzhou)
 #   APT_MIRROR        - APT mirror host (default: empty; set to mirrors.aliyun.com for China)
+#   OPENCLAW_REPO_URL - OpenClaw git repository URL (default: https://github.com/johnlanni/openclaw.git)
 # ============================================================
 
 ARG HIGRESS_REGISTRY=higress-registry.cn-hangzhou.cr.aliyuncs.com
@@ -24,6 +25,7 @@ ARG HTTPS_PROXY
 ARG http_proxy
 ARG https_proxy
 ARG NPM_REGISTRY=https://registry.npmjs.org/
+ARG OPENCLAW_REPO_URL=https://github.com/johnlanni/openclaw.git
 
 # Set proxy as environment variables to ensure all tools can use them
 ENV HTTP_PROXY=${HTTP_PROXY} \
@@ -52,7 +54,7 @@ RUN curl -fsSL https://deb.nodesource.com/setup_22.x | bash - && \
 # 2. Install dependencies and build
 # 3. Remove unnecessary files (tests, .git, etc.) - keep all node_modules for extensions
 RUN git clone --depth 1 --single-branch -b hiclaw-v1 \
-        https://github.com/johnlanni/openclaw.git /opt/openclaw && \
+        ${OPENCLAW_REPO_URL} /opt/openclaw && \
     cd /opt/openclaw && \
     git fetch --depth 1 origin 86dad1383c12629ad3aa48575f2dd350fc0775d4 && \
     git checkout 86dad1383c12629ad3aa48575f2dd350fc0775d4 && \

